xmipptomo/protocols.conf

```diff
@@ -4,6 +4,7 @@ Tomography = [
 	{"tag": "section", "text": "Movies/Micrographs", "children": []},
 	{"tag": "section", "text": "Reconstruction", "children": []},
     {"tag": "section", "text": "Particles", "children": [
+        {"tag": "protocol", "value": "XmippProtPhantomSubtomo", "text": "default"},
```

Comment:
I like this protocol for testing purposes...but I'm not sure if we want to expose it in the "tree". Maybe is enough to appear when doing "Ctrl +F"?

Reply:
I was also not sure about where to put this protocol in the tree. I decided to put it there because the output is an "initial" setOfSubtomograms ready to be aligned (for example) as if they were already imported or picked. But I accept any suggestions. I think it could be interesting to have this protocol exposed to users as I know some cases that users wants to create a phantom to test something or as a "very initial" model. But I certainly don't know if that is a common practise or just few cases. However I consider that give options to users it could be always usefull for them.

Reply:
I agree with the "give options to users". What about a dedicated section: "Tests"?

Reply:
As you consider it. Where do you think section "tests" should be in the tree? at the end?

Reply:
Done in https://github.com/I2PC/scipion-em-xmipptomo/pull/8/commits/7dee3d25d74c4b073a75072fc7baac7ca7aa7c8d